src/app/styles/_variables.scss

```diff
@@ -23,17 +23,17 @@ $grid-breakpoints: (
   sm: $grid-sm,
   md: $grid-md,
   lg: $grid-lg,
-  xl: $grid-xl
+  xl: $grid-xl,
 );
 
-$container-sm: 540px;
-$container-md: 720px;
-$container-lg: 960px;
-$container-xl: 1140px;
+$container-sm: 576px;
+$container-md: 768px;
+$container-lg: 992px;
+$container-xl: 1400px;
```

Comment:
there a reason for these changes?

i think customising here has the potential to cause heaps of issues

Reply:
Mostly bringing the container sizes in-line with the grid. I also expanded the large container because its very limiting on our website.